src/ToricVarieties/CyclicQuotientSingularities/CyclicQuotientSingularities.jl

```diff
@@ -72,7 +72,12 @@ julia> ecf = cf[1]-1//(cf[2]-fmpq(1,cf[3]))
 7//5
 ```
 """
-continued_fraction_hirzebruch_jung(cqs::CyclicQuotientSingularity) = Vector{fmpz}(pm_object(cqs).CONTINUED_FRACTION)
+function continued_fraction_hirzebruch_jung(cqs::CyclicQuotientSingularity)
+    if !has_attribute(cqs, :continued_fraction_hirzebruch_jung)
+        set_attribute!(cqs, :continued_fraction_hirzebruch_jung, Vector{fmpz}(pm_object(cqs).CONTINUED_FRACTION))
+    end
+    return get_attribute(cqs, :continued_fraction_hirzebruch_jung)
```

Comment:
Thank you. That is a good suggestion. Indeed, this change can be done in a ton of places. I will do this in a new commit rather than rebasing everything (unless I hear differently).

Reply:
Adjusted in https://github.com/oscar-system/Oscar.jl/pull/839/commits/5a3702598bcf6f3ce372e6edc86121bfeccbf028.